Cargo.toml

```
@@ -78,7 +78,7 @@ aws-config = "1.2.0"
 aws-credential-types = "1.2.0"
 aws-sdk-dynamodb = "1.38.0"
 aws-sdk-s3 = "1.38.0"
-half = { "version" = "2.4.1", default-features = false, features = [
+half = { "version" = "^2.5", default-features = false, features = [
```

Could we just do 2.1 [like arrow pins](https://crates.io/crates/arrow/54.3.1/dependencies)? This is more flexible and I don't think we require any new features of half.
```suggestion
half = { "version" = "2.1", default-features = false, features = [
```